Troubleshooting/statedump.md

```diff
@@ -16,14 +16,23 @@ Run the command
 
                 gluster --print-statedumpdir
 
-to find out which directory the statedumps will be created in. This directory may need to be created if not already present.
-In this document, we will refer to this directory as `statedump-directory`.
+on a gluster server node to find out which directory the statedumps will be created in. This directory may need to be created if not already present.
+For the rest of this document, we will refer to this directory as `statedump-directory`.
```

Comment:
There are no more references to `statedump-directory` in this doc anymore.

Reply:
Fixed.